DZ/DZ/ViewControllerFour.swift

```diff
@@ -0,0 +1,238 @@
+//
+//  ViewControllerFour.swift
+//  lession7 dz
+//
+//  Created by Сережа on 23.05.2023.
+//
+
+import UIKit
+
+class ViewControllerFour: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureSubviews()
+        createToolbar()
+    }
+    
+    lazy var brendLabel: UILabel = {
+       var brendLabel = UILabel()
+        brendLabel.frame = CGRect(x: 30, y: 30, width: 150, height: 30)
+        brendLabel.font = brendLabel.font.withSize(20)
+        brendLabel.textColor = .systemGray
+        brendLabel.text = "Марка авто"
+        return brendLabel
+    }()
+    
+    lazy var brendTextField: UITextField = {
+       var brendTextField = UITextField()
+        brendTextField.frame = CGRect(x: 30, y: 70, width: 150, height: 30)
+        brendTextField.borderStyle = UITextField.BorderStyle.roundedRect
+        brendTextField.tintColor = UIColor.systemGray
+
+        return brendTextField
+    }()
+    
+    lazy var modelLabel: UILabel = {
+       var modelLabel = UILabel()
+        modelLabel.frame = CGRect(x: 30, y: 110, width: 150, height: 30)
+        modelLabel.font = modelLabel.font.withSize(20)
+        modelLabel.textColor = .systemGray
+        modelLabel.text = "Модель авто:"
+        return modelLabel
+    }()
+    
+     lazy var modelTextField: UITextField = {
```

Comment:
```suggestion
   private  lazy var modelTextField: UITextField = {
```
все элементы визуальные приватны